build

```diff
@@ -11,8 +11,8 @@ GLDFLAGS=${GLDFLAGS:-}
 export GOFLAGS=-mod=vendor
 
 if [ -z ${VERSION+a} ]; then
-	echo "Using version from git..."
 	VERSION=$(git describe --dirty --always)
+	echo "Using version from git: $VERSION"
```

Comment:
Looks like some of the prep bits could be factored out into a separate PR/commit.